rfcs/text/0023-match_only_text-data-type.md

```diff
@@ -84,12 +233,33 @@ Stage 2: Identifies scope of impact of changes. Are breaking changes required? S
 The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
 -->
 
+Ingestion mechanisms will need to adopt the new mappings to index fields as the new type. Since both `text` and `match_only_text` are members of the `text` type family, this will not cause a conflict in Kibana index patterns across indices using the two types for the same field.
+
+As with all type changes in ECS, the ECS team will benchmark and identify any significant storage, indexing, or query changes. The Elasticsearch performance will be engaged, if necessary, to determine the results. However, `match_only_text` is a `text` field with these settings:
+
+* `index_options: docs`
+* `norms: false`
+
+As a convention, ECS already sets `norms: false` on all `text` fields, so this setting should have no impact on performance. Setting `index_options: docs` only indexes the doc number and has already been recommended to reduce the disk usage needed for indexing.
+
+Negative performance or storage side-effects from this change are not expected.
```

Comment:
Except for phrase and intervals queries. I know it's stated a couple lines below, but taken out of context this sentence could suggest that everything is faster or as fast.

Reply:
Rephrased this line to note this as a known limitation.